cmd/training-operator.v1/main.go

```diff
@@ -52,6 +52,8 @@ import (
 const (
 	// EnvKubeflowNamespace is an environment variable for namespace when deployed on kubernetes
 	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
+
+	webhookConfigurationName = "validator.training-operator.kubeflow.org"
```

Comment:
IIUC, we need to refine the webhook controller-gen marker as well: https://github.com/kubeflow/training-operator/blob/6965c1a92462d46981071748936eb135a4584f3d/pkg/webhooks/pytorch/pytorchjob_webhook.go#L51

Reply:
I don't think we should change it, since those parameters defines the particular webhook configuration, not the name of validation webhook configuration object. For example: 
https://github.com/kubeflow/training-operator/blob/4bab32ab8898f33fbae45c694165afce69257c32/manifests/v2/base/webhook/manifests.yaml#L35

Reply:
Ah, you're right.
This will be used in 

```yaml
- op: replace
  path: /metadata/name
  value: validator.training-operator-v2.kubeflow.org
```